[2024-06-06T03:21:51.861+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-06T03:21:52.000+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_testConnection_v1.test_first_connection scheduled__2023-03-06T00:00:00+00:00 [queued]>
[2024-06-06T03:21:52.070+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_testConnection_v1.test_first_connection scheduled__2023-03-06T00:00:00+00:00 [queued]>
[2024-06-06T03:21:52.080+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-06T03:21:52.191+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): test_first_connection> on 2023-03-06 00:00:00+00:00
[2024-06-06T03:21:52.238+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=702) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-06T03:21:52.268+0000] {standard_task_runner.py:63} INFO - Started process 725 to run task
[2024-06-06T03:21:52.254+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_testConnection_v1', 'test_first_connection', 'scheduled__2023-03-06T00:00:00+00:00', '--job-id', '142', '--raw', '--subdir', 'DAGS_FOLDER/TestConnectionDag.py', '--cfg-path', '/tmp/tmpvamk6_zn']
[2024-06-06T03:21:52.307+0000] {standard_task_runner.py:91} INFO - Job 142: Subtask test_first_connection
[2024-06-06T03:21:52.818+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_testConnection_v1.test_first_connection scheduled__2023-03-06T00:00:00+00:00 [running]> on host fe4af7f6193a
[2024-06-06T03:21:54.522+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_testConnection_v1' AIRFLOW_CTX_TASK_ID='test_first_connection' AIRFLOW_CTX_EXECUTION_DATE='2023-03-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-03-06T00:00:00+00:00'
[2024-06-06T03:21:54.561+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-06T03:21:54.822+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-06T03:21:54.834+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: <lambda>() missing 1 required positional argument: '_'
[2024-06-06T03:21:54.912+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=dag_testConnection_v1, task_id=test_first_connection, run_id=scheduled__2023-03-06T00:00:00+00:00, execution_date=20230306T000000, start_date=20240606T032152, end_date=20240606T032154
[2024-06-06T03:21:55.288+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 142 for task test_first_connection (<lambda>() missing 1 required positional argument: '_'; 725)
[2024-06-06T03:21:55.442+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-06T03:21:55.583+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
